packages/dds/tree/src/test/snapshots/snapshotTools.ts

```diff
@@ -47,7 +47,8 @@ let currentTestFile: string | undefined;
 // Simple filter to avoid tests with a name that would accidentally be parsed as directory traversal or other confusing things.
 const nameCheck = new RegExp(/^[^"/\\]+$/);
 
-assert(__dirname.endsWith("dist/test/snapshots"));
+console.log(`__dirname: ${__dirname}`);
+assert(__dirname.endsWith("dist/test/snapshots") || __dirname.endsWith("dist\\test\\snapshots"));
```

Comment:
You could also use a regex here,

```suggestion
assert(__dirname.match(/dist[/\\]test[/\\]snapshots$/)
```

Though I don't think one way is strictly better than the other